desktop-app/app/actions/browser.js

```diff
@@ -85,6 +88,27 @@ export function newInspectorState(status) {
   };
 }
 
+export function newCSSEditorState(status) {
+  return {
+    type: NEW_CSS_EDITOR_STATUS,
+    status,
+  };
+}
+
+export function newCSSEditorPosition(position) {
+  return {
+    type: NEW_CSS_EDITOR_POSITION,
+    position,
+  };
+}
+
+export function newCSSEditorContent(content) {
```

Comment:
@sidthesloth92 I'm in the favor of keeping component states within the components. 
But here I have pushed it to `redux` state so that it is persisted across the editor sessions. Meaning the user can close the editor and contents will be there persisted when they open it again.